src/views/wallet/WalletView.tsx

```diff
@@ -34,15 +34,23 @@ const WalletView: React.FC = (): JSX.Element => {
   // we have to add 'undefined'  as default value
   const keystore = useObservableState(keystoreService.keystore$, undefined)
 
+  const reloadButton = useMemo(
+    () => (
+      <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
```

Comment:
We have to avoid inline styling since we're using `styled-components`.

Reply:
yep. you are right here. but i did not want to add another HOC here for just alignments

Reply:
@thatStrangeGuyThorchain You could use `Row` from `antd` here - no need for another styled component.